### DISCUSSION TITLE
FR: Enable custom fields for commerce relation field mapping

### DISCUSSION THREAD

**Discussion by ZapaNOR** [engram-design/FeedMe#902] [Ideas]
When importing an entry relation field, you can chose what source this fields is mapped too, including custom fields on the entry.

![Screenshot 2019-08-09 at 12 08 43](https://user-images.githubusercontent.com/53942798/62781901-46bcb880-bab9-11e9-8545-fe9d530925fc.png)

When importing a commerce product relation field you only get the option to map against Title, ID or slug.

![Screenshot 2019-08-09 at 12 08 30](https://user-images.githubusercontent.com/53942798/62781912-4cb29980-bab9-11e9-9268-4e5f40b2322c.png)

I'm trying to import products from an external system, these have a relation on the SKU of that product. Since I cannot use custom fields (or sku) to map related products, there is currently no way for me to create these relations without using SKU in the title or slug.

**Comment by rrr3da** [comment]
any updates on this?
thank you

**Comment by angrybrad** [comment]
Feed Me [6.13.0](https://github.com/craftcms/feed-me/releases/tag/6.13.0) is out with this!
